backend/openagent/index/pgvector_store.py

```diff
@@ -0,0 +1,36 @@
+from dotenv import load_dotenv
+from langchain.embeddings import OpenAIEmbeddings
+from langchain_core.documents import Document
+from langchain_postgres.vectorstores import PGVector
+
+from openagent.conf.env import settings
+
+load_dotenv()
+
+
+def build_vector_store() -> PGVector:
```

Comment:
can we add generic support for other vector databases?

Reply:
any subclass implement ` Vector Store(langchain_core/vectorstores.py)` should be ok